src/schema/slack/functions/_scripts/src/template.ts

```diff
@@ -0,0 +1,149 @@
+import { pascalCase } from "https://deno.land/x/case@v2.1.0/mod.ts";
+import SchemaTypes from "../../../../schema_types.ts";
+import SlackSchemaTypes from "../../../schema_types.ts";
+import { FunctionParameter, FunctionRecord } from "./types.ts";
+
+type StringObject = Record<string, string>;
+
+const autogeneratedComment = () => {
+  const time = new Date();
+  return `/** This file was autogenerated on ${time.toDateString()}. Run ./.slack/sdk/schema/slack/functions/_scripts/generate to rebuild **/`;
```

Comment:
I am confused by the path provided in this error hint. Why is it recommending to run the script from a `.slack/` directory?

Reply:
Oh good catch! This is something I copied over, I'll rewrite it. Tbh I don't think it matters what your `cwd` is when you run the script, so you can follow the README and move to the `_scripts/` directory to run `./generate` or you can run `src/schema/slack/functions/_scripts/generate` from the project root.